Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1701,6 +1701,8 @@ def lhegensim(fragment,howMuch):
 defaultDataSets={}
 defaultDataSets['2017']='CMSSW_8_1_0_pre16-81X_upgrade2017_realistic_v22-v'
 defaultDataSets['2017Design']='CMSSW_8_1_0_pre16-81X_upgrade2017_design_IdealBS_v6-v'
+defaultDataSets['2018']='CMSSW_8_1_0_pre16-81X_upgrade2017_realistic_v22-v'
+defaultDataSets['2018Design']='CMSSW_8_1_0_pre16-81X_upgrade2017_design_IdealBS_v6-v'
```

Comment:
These GEN-SIM were made with the phase1 HE
and won't be consistent with the re-defined 2017 scenario